tests/components/nobo_hub/test_config_flow.py

```diff
@@ -0,0 +1,260 @@
+"""Test the Nobø Ecohub config flow."""
+from unittest.mock import Mock, PropertyMock, patch
+
+from pynobo import nobo
+
+from homeassistant import config_entries, setup
+from homeassistant.components.nobo_hub.const import (
+    CONF_OVERRIDE_TYPE,
+    CONF_WEEK_PROFILE_NONE,
+    DOMAIN,
+)
+from homeassistant.const import CONF_COMMAND_OFF, CONF_COMMAND_ON
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_configure_with_ip(hass: HomeAssistant) -> None:
+    """Test configuring with IP address."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    with patch(
+        "pynobo.nobo.async_discover_hubs",
+        return_value=[],
+    ):
+        result = await hass.config_entries.flow.async_init(
+            DOMAIN, context={"source": config_entries.SOURCE_USER}
+        )
+        assert result["type"] == "form"
+        assert result["errors"] == {}
+
+        with patch(
```

Comment:
My impression is that the first patch for discover hubs is only needed on the init step, then the rest of the code does not need to be indent inside of it. That is, i think this can be unindented by one block.

Reply:
You are quite correct.

Reply:
Fixed in updated tests.